src/main/java/org/opensearch/security/tools/democonfig/Installer.java

```diff
@@ -205,23 +205,28 @@ static void setOpenSearchVariables() {
         OPENSEARCH_LIB_PATH = BASE_DIR + "lib" + File.separator;
         OPENSEARCH_INSTALL_TYPE = determineInstallType();
 
+        boolean shouldExit = false;
         if (!(new File(OPENSEARCH_CONF_FILE).exists())) {
-            System.out.println("Unable to determine OpenSearch config directory. Quit.");
-            System.exit(-1);
+            System.out.println("Unable to determine OpenSearch config file. Quit.");
+            shouldExit = true;
```

Comment:
Nit: better control flow by using a collection to captures validation error, and then if you have any errors exiting